src/main/java/com/cocodan/triplan/schedule/controller/ScheduleController.java

```diff
@@ -64,23 +67,39 @@
     // 메모
     @PostMapping("/{scheduleId}/memos")
     public ResponseEntity<Long> createMemo(@PathVariable Long scheduleId, @RequestBody @Valid MemoRequest memoRequest) {
-        Member member = (Member) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+        Member member = getMember();
         Long savedId = scheduleService.createMemo(scheduleId, memoRequest, member.getId());
 
         return new ResponseEntity<>(savedId, HttpStatus.CREATED);
     }
 
+    @GetMapping("/{scheduleId}/memos")
+    public ResponseEntity<List<MemoSimpleResponse>> getMemos(@PathVariable Long scheduleId) {
+        Member member = getMember();
+        List<MemoSimpleResponse> memos = scheduleService.getMemos(scheduleId, member.getId());
+
+        return new ResponseEntity<>(memos, HttpStatus.OK);
```

Comment:
```suggestion
        return ResponseEntity.ok(memos);
```
ok 처럼 별도로 구현되어 있는 메서드들 사용하는거 어떻게 생각하시나요?

Reply:
와 이건 처음 알았어요. 굿굿굿

Reply:
이건 나중에 한 번에 반영하겠습니다.